src/GRA.Controllers/ViewComponents/DisplayNotificationsViewComponent.cs

```diff
@@ -77,6 +77,10 @@ public IViewComponentResult Invoke()
                         notification.AttachmentFilename
                             = _pathResolver.ResolveContentPath(notification.AttachmentFilename);
                     }
+                    if (HttpContext.Items[ItemKey.AskToAddFamilyMember].ToString() == "True")
+                    {
+                        notification.SupplementalText = $"Would you like to <a href='{Url.Action(nameof(ProfileController.AddHouseholdMember), ProfileController.Name)}'>add a family member?</a>";
```

Comment:
This needs to be localized properly.

Reply:
I thought as much. FYI there are other notifications that go through that component that aren't getting localized currently:
![Screenshot 2024-01-09 164725](https://github.com/MCLD/greatreadingadventure/assets/12805821/6fa393ab-2b87-48f3-9ac8-28c1172caf47)
![Screenshot 2024-01-09 164850](https://github.com/MCLD/greatreadingadventure/assets/12805821/16bce919-7a52-4587-9943-041e6f9a43f7)

Reply:
Opened an issue for these two:  #1082